account_wallet/views/wallet_views.xml

```diff
@@ -0,0 +1,172 @@
+                            <field name="partner_id" />
+                            <field name="wallet_type_id" />
+                            <field name="company_currency_id" invisible="1" />
+                            <field
+                            name="balance"
+                            widget="monetary"
+                            options="{'currency_field': 'company_currency_id'}"
+                        />
+                            <field name="active" invisible="1" />
+                        </group>
+                    </sheet>
+                    <div class="oe_chatter">
+                        <field name="message_follower_ids" groups="base.group_user" />
+                        <field name="activity_ids" />
+                        <field name="message_ids" />
+                    </div>
+                </form>
+           </field>
+        </record>
+
+        <record id="account_wallet_action" model="ir.actions.act_window">
+            <field name="name">Wallet</field>
+            <field name="type">ir.actions.act_window</field>
+            <field name="res_model">account.wallet</field>
+            <field name="view_mode">tree,form</field>
+            <field name="context">{'search_default_wallet_type_id':1}</field>
+        </record>
+
+        <menuitem
+        id="account_wallet_parent_menu"
+        name="Wallet"
+        parent="account.menu_finance"
+        sequence="4"
+    />
+
+        <menuitem
+        action="account_wallet_action"
+        id="account_wallet_menu"
+        parent="account_wallet_parent_menu"
+        sequence="10"
+    />
+
+
+        <!-- Account move line inherit -->
+        <record id="account_move_line_view_tree_inherit_wallet_base" model="ir.ui.view">
+            <field name="name">account.move.line.tree (in account_wallet)</field>
+            <field name="model">account.move.line</field>
+            <field name="inherit_id" ref="account.view_move_line_tree" />
+            <field name="arch" type="xml">
+                <field name="statement_id" position="after">
+                    <field name="account_wallet_id" />
+                </field>
+            </field>
+        </record>
+
+        <record id="account_move_view_form_inherit_wallet_base" model="ir.ui.view">
+            <field name="name">account.move.form (in account_wallet)</field>
+            <field name="model">account.move</field>
+            <field name="inherit_id" ref="account.view_move_form" />
+            <field name="arch" type="xml">
+                <data>
+                    <xpath
+                    expr="//notebook/page/field[@name='line_ids']/tree/field[@name='date_maturity']"
+                    position="after"
+                >
+                        <field name="account_wallet_id" />
+                    </xpath>
+                </data>
+            </field>
+        </record>
+
+        <record
+        id="account_move_line_view_form_inherit_cagnotte_base"
```

Comment:
some "cagnotte" are remaing. (if I understood correctly, you wanted to rename cagnotte into wallet).

3 occurences in this PR.

![image](https://user-images.githubusercontent.com/3407482/179491238-e483e3eb-8442-4827-bca8-5e9bb48c59c1.png)